tck/bridge-tck-main-portlet/src/main/webapp/tests/resourcesRenderedInHeadTest.xhtml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<f:view xmlns="http://www.w3.org/1999/xhtml" xmlns:f="http://xmlns.jcp.org/jsf/core"
+		xmlns:h="http://xmlns.jcp.org/jsf/html" xmlns:portlet="http://xmlns.jcp.org/portlet_3_0"
+		xmlns:c="http://java.sun.com/jsp/jstl/core">
+	<h:head>
+		<h:outputScript library="test" name="resourcesRenderedInHeadTest.js" />
+		<portlet:resourceURL var="testResource1">
+			<portlet:param name="ln" value="test" />
+			<portlet:param name="javax.faces.resource" value="resource1.js" />
+		</portlet:resourceURL>
+		<script type="text/javascript" src="${testResource1}"></script>
+		<h:outputStylesheet library="test" name="resource1.css" />
+	</h:head>
+	<h:body>
+		<h:outputScript target="head">
+			// Set the id of this script since h:outputScript's id attribute is not rendered in the html.
+			var scripts = document.getElementsByTagName('script');
+			var currentScript = scripts[scripts.length - 1];
```

Comment:
@stiemannkj1 Portlet containers other than Pluto may include `<script>` elements as part of a skin/theme that could cause this JS to break. Please send a follow-up PR with a refactor. Perhaps there is a way to look for specific text as a child of the `<script>` in order to identify it?